init/systemd/ipa.sysusers.base.conf

```diff
@@ -0,0 +1,5 @@
+u kdcproxy  -   "IPA KDC Proxy User"    -
+u dirsrv    -   -   -
```

Comment:
@dkupka ```create_system_user()``` adds GECOS for DS and PKI user.

https://github.com/freeipa/freeipa/pull/697/files#diff-39163021c252126be7c9fbc0c66b1668L452